components/admin-panel/sections/ReceiptTemplateForm.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Flex } from '../../Grid';
+import PreviewModal from '../../PreviewModal';
+import StyledButton from '../../StyledButton';
+import StyledInput from '../../StyledInput';
+import StyledTextarea from '../../StyledTextarea';
+import { Label, P, Span } from '../../Text';
+
+const ReceiptTemplateForm = ({ defaultTemplate, value, onChange, placeholders }) => {
+  const [showPreview, setShowPreview] = React.useState(false);
+
+  return (
+    <React.Fragment>
+      <Label htmlFor="receipt-title" color="black.800" fontSize="16px" fontWeight={700} lineHeight="24px">
+        <FormattedMessage defaultMessage="Receipt title" />
+      </Label>
+      <StyledInput
+        id="receipt-title"
+        placeholder={placeholders.title}
+        defaultValue={value.title}
+        onChange={e => onChange(e.target.value, null)}
+        width="100%"
+        maxWidth={414}
+        mt="6px"
+      />
+      {defaultTemplate && (
```

Comment:
From the parent, it's not clear that this prop is going to be a boolean and what impact it's going to have on the component. I suggest:

```suggestion
      {showTipForDefaultTitle && (
```